kube_log_watcher/templates/scalyr.json.jinja2

```diff
@@ -18,6 +18,7 @@
         {% for log in logs %}
             {
                 "path": "{{ log.path }}",
+                "rename_logfile": "application={{ log.attributes.application | quote_plus }}/component={{ log.attributes.component | quote_plus }}/version={{ log.attributes.version | quote_plus }}",
```

Comment:
How about adding of some well-known prefix to simplify parsing? Something like `scalyr_kv_renamed/`?

Reply:
Won't we have to use some regex like this (^application=.*/component=.*/version=.*$) to detect this special format anyway?
So the format acts as some kind of magic token already?

Reply:
I'm for using more flexible format and parser. What if we'll decide to add or remove some part from this filename.
I thought to implement it as "if path is start with special prefix, split items by `/` and keys-values by `=`". Without any bind to property names.

Reply:
As discussed we will use question mark as prefix (similar to query-string in URL)

Reply:
Don't you want to use http://jinja.pocoo.org/docs/2.10/templates/#urlencode ?

Reply:
Discussed and decided to not using this to keep defined parameters order